SUILesson5/SUILesson5/ContentView.swift

```diff
@@ -7,15 +7,205 @@
+
+            VStack {
+                formWithCurrentCost
+                textIwthLotName
+                Spacer()
+                zstackWithImage
+                vstackWithPickers
+                textWithOffersCost
+                offersCostSlider
+                vstackWithBetButtons
+            }
+            .animation(.spring())
+            .offset(y: CGFloat(vstackOffset))
+        }
+    }
+
+    var stackWithInfoAboutAuto: some View {
         VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
+            Text(Constants.haracteristicsName)
+            HStack {
+                Text(Constants.valueOfEngineName).padding()
+                Spacer()
+                Text(Constants.valueOfEngine).padding()
+            }
+            HStack {
+                Text(Constants.acselerateForHundredName).padding()
+                Spacer()
+                Text(Constants.acselerateForHundredValue).padding()
+            }
+            HStack {
+                Text(Constants.horsePowerName).padding()
+                Spacer()
+                Text(Constants.horsePowerValue).padding()
+            }
+            HStack {
+                Text(Constants.masseName).padding()
+                Spacer()
+                Text(Constants.masseValue).padding()
+            }
+            HStack {
+                Text(Constants.consumptionName).padding()
+                Spacer()
+                Text(Constants.consumptionValue).padding()
+            }
+            Button(Constants.backToLot) {
+                self.vstackOffset = 0
+                self.hstackOffset = -1000
+            }
+        }
+    }
+
+    var formWithCurrentCost: some View {
+        Form {
+            Text("\(Constants.currentCostName)\(currentPrice + Int(viewModel.currentLotPrice)) \(Constants.rublesName)")
+        }
+    }
+
+    var textIwthLotName: some View {
+        Text("Лот № \(segmentIndexCarsModel + 1) - \(viewModel.company[segmentIndexCompany].name)").font(.system(.largeTitle))
+    }
+
+    var zstackWithImage: some View {
+        ZStack {
+            RoundedRectangle(cornerRadius: 10)
+                .fill(Color.gray)
+                .offset(x: CGFloat(offsetX))
+                .padding()
+
+            Image(viewModel.currentCompany.models[segmentIndexCarsModel])
+                .resizable()
+                .scaledToFit()
+                .frame(width: 300, height: 300)
+                .offset(x: CGFloat(offsetX))
+        }
+    }
+
+    var vstackWithPickers: some View {
+        VStack {
+            Picker(selection: Binding(get: {
+                self.segmentIndexCompany
+            }, set: { (newValue) in
+                self.segmentIndexCompany = newValue
+                self.viewModel.setCurrentCompany(tag: newValue)
+            }), label: Text(Constants.emptyString)) {
+                ForEach(0..<3) {
```

Comment:
```suggestion
                ForEach(0..<3) {
```
тут почему берешь 3,а не массив .count?